tests/ttnn/unit_tests/operations/test_maxpool2d.py

```diff
@@ -187,6 +189,10 @@ def run_max_pool(
             # [64, 16, 528, 80],  ## oom
             # [128, 16, 528, 80], ## oom
             # [256, 16, 528, 80], ## oom
+            ## wide for vgg
+            [1, 256, 56, 56],
+            [1, 512, 28, 28],
+            [1, 512, 14, 14],
```

Comment:
is it possible to test it on multiple batches too?
also what if W or H is not equal to each other.
Also the case wher W or H is equal to 1 is interesting too.

Reply:
These are specific to what are used in models -- the generic testing will be all in sweep tests.

Reply:
But if it works on other cases why not demonstrate it here?